src/material/tabs/tab.ts

```diff
@@ -79,6 +79,40 @@ export class MatTab extends _MatTabBase implements OnInit, CanDisable, OnChanges
    */
   @Input('aria-labelledby') ariaLabelledby: string;
 
+  /**
+   * Takes classes set on the host mat-tab element and applies them to the tab
+   * label inside the mat-tab-header container to allow for easy styling.
+   */
+  @Input('class')
+  set labelClass(value: string | string[]) {
+    if (value && value.length) {
```

Comment:
I don't think that we need all of this coercion. The property can just be `@Input() labelClass: string | string[]` since it gets passed in directly to `ngClass`.